src/main/java/com/github/tomakehurst/wiremock/client/WireMock.java

```diff
@@ -33,34 +33,7 @@
 import com.github.tomakehurst.wiremock.http.DelayDistribution;
 import com.github.tomakehurst.wiremock.http.Request;
 import com.github.tomakehurst.wiremock.http.RequestMethod;
-import com.github.tomakehurst.wiremock.matching.AbsentPattern;
-import com.github.tomakehurst.wiremock.matching.AfterDateTimePattern;
-import com.github.tomakehurst.wiremock.matching.BeforeDateTimePattern;
-import com.github.tomakehurst.wiremock.matching.BinaryEqualToPattern;
-import com.github.tomakehurst.wiremock.matching.ContainsPattern;
-import com.github.tomakehurst.wiremock.matching.EqualToDateTimePattern;
-import com.github.tomakehurst.wiremock.matching.EqualToJsonPattern;
-import com.github.tomakehurst.wiremock.matching.EqualToPattern;
-import com.github.tomakehurst.wiremock.matching.EqualToXmlPattern;
-import com.github.tomakehurst.wiremock.matching.ExactMatchMultiValuePattern;
-import com.github.tomakehurst.wiremock.matching.IncludesMatchMultiValuePattern;
-import com.github.tomakehurst.wiremock.matching.LogicalAnd;
-import com.github.tomakehurst.wiremock.matching.LogicalOr;
-import com.github.tomakehurst.wiremock.matching.MatchesJsonPathPattern;
-import com.github.tomakehurst.wiremock.matching.MatchesXPathPattern;
-import com.github.tomakehurst.wiremock.matching.MultiValuePattern;
-import com.github.tomakehurst.wiremock.matching.MultipartValuePatternBuilder;
-import com.github.tomakehurst.wiremock.matching.NegativeContainsPattern;
-import com.github.tomakehurst.wiremock.matching.NegativeRegexPattern;
-import com.github.tomakehurst.wiremock.matching.NotPattern;
-import com.github.tomakehurst.wiremock.matching.RegexPattern;
-import com.github.tomakehurst.wiremock.matching.RequestPattern;
-import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
-import com.github.tomakehurst.wiremock.matching.StringValuePattern;
-import com.github.tomakehurst.wiremock.matching.UrlPathPattern;
-import com.github.tomakehurst.wiremock.matching.UrlPathTemplatePattern;
-import com.github.tomakehurst.wiremock.matching.UrlPattern;
-import com.github.tomakehurst.wiremock.matching.ValueMatcher;
+import com.github.tomakehurst.wiremock.matching.*;
```

Comment:
🔴 It actually violates Google Java Style Guide: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements